components/images/responsive-images.css

```diff
@@ -0,0 +1,38 @@
+@import url(/css/base.css);
+@import url(/css/utility.css);
+
+/* reponsive images start here */
+.et_ri-container{
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+.et_ri-container>*{
+    background-color: var(--white-color);
+    border: 1px solid var(--light-gray);
+    margin: 2rem;
+    padding: 2rem 0rem;
+    text-align: center;
+}
+/* simple responsive image start here */
+.et_simple-ri{
+    width: 80%;
+}
+.et_simple-ri img{
+    vertical-align:bottom;
+    max-width: 60%;
+}
+/* simple responsive image end here */
+/* rounded responsive image start here */
+.et_rounded-ri{
+    padding: 1rem;
+}
+.et_rounded-ri img{
+    clip-path: circle();
+    min-width: 50px;
```

Comment:
Please use rem

Reply:
> You can remove the inputs.js file mentioned in the inputs folder if it isn't required. Rest of the code looks good to me. Good work on the Images and Input component Namrata.

yes, will remove it. Thankyou so much for reviewing!!